subnet/kube/kube.go

```diff
@@ -253,7 +266,13 @@ func (ksm *kubeSubnetManager) GetNetworkConfig(ctx context.Context) (*subnet.Con
 // with importany information for the backend, such as the subnet. This function is called once by the backend when
 // registering
 func (ksm *kubeSubnetManager) AcquireLease(ctx context.Context, attrs *subnet.LeaseAttrs) (*subnet.Lease, error) {
-	cachedNode, err := ksm.nodeStore.Get(ksm.nodeName)
+	var cachedNode *v1.Node
+	var err error
+	if ksm.disableNodeInformer {
+		cachedNode, err = ksm.client.CoreV1().Nodes().Get(ctx, ksm.nodeName, metav1.GetOptions{ResourceVersion: "0"})
```

Comment:
Do you forget to add clusterrole for this operation ?

https://github.com/flannel-io/flannel/blob/9184f549b53d7c999b96a99180c9ba3aa873a79f/Documentation/kube-flannel.yml#L22-L26


@xh4n3 

/cc  @rbrtbnfgl @manuelbuil

Reply:
@JornShen Yes, I didn't notice there were YAMLs in this repo. 😭 

There are quite a lot YAMLs in the Documentation directory. Which YAMLs should I update?  @rbrtbnfgl 

```bash
./k8s-manifests/kube-flannel-legacy.yml
./k8s-manifests/kube-flannel-rbac.yml
./kube-flannel-aliyun.yml
./kube-flannel.yml
./kube-flannel-old.yaml
```

Reply:
The latest one should be `kube-flannel.yml`

Reply:
@rbrtbnfgl @JornShen just opened a pr regarding this issue, https://github.com/flannel-io/flannel/pull/1677